scrapy/exporters.py

```diff
@@ -96,9 +100,13 @@ class JsonItemExporter(BaseItemExporter):
     def __init__(self, file, **kwargs):
         self._configure(kwargs, dont_fail=True)
         self.file = file
-        self.encoder = ScrapyJSONEncoder(**kwargs)
+        self.encoder = ScrapyJSONEncoder(ensure_ascii=self.ensure_ascii, **kwargs)
         self.first_item = True
 
+    def _configure(self, options, dont_fail=False):
+        self.ensure_ascii = options.pop('ensure_ascii', True)
```

Comment:
Regarding feed encoding setting,
I believe we could to use `None` as default feed encoding,

* for JSON that would mean `ensure_ascii=True`, i.e. `\uXXXX` escape sequences
* for XML and CSV, that would mean UTF-8

if feed encoding set to UTF-8 explicitly:

* for JSON export, that would mean `ensure_ascii=False` and UTF-8 encoding
* for XML and CSV, that would change from default `None` setting

For other encodings, data is written with the supplied encoding, with warnings or exceptions for failing item encoding writes.
what do you think?